R/tm_g_patient_profile.R

```diff
@@ -698,8 +699,8 @@ srv_g_patient_profile <- function(id,
                     ) %>%
                     select(USUBJID, ASTDT, AENDT, ASTDY, AENDY, !!quo(cm_var))
                   if (length(unique(ADCM$USUBJID)) > 0) {
-                    ADCM <- ADCM[which(ADCM$AENDY >= -28 | is.na(ADCM$AENDY) == TRUE # nolint
-                    & is.na(ADCM$ASTDY) == FALSE), ] # nolint
+                    ADCM <- ADCM[which(ADCM$AENDY >= -28 | is.na(ADCM$AENDY) == TRUE &
+                      is.na(ADCM$ASTDY) == FALSE), ]
```

Comment:
This is failing locally, it can take a single line as it's under 120 characters

```suggestion
                    ADCM <- ADCM[which(ADCM$AENDY >= -28 | is.na(ADCM$AENDY) == TRUE & is.na(ADCM$ASTDY) == FALSE), ]
```

Reply:
Styler did it :angry:

Reply:
It was passing on my machine but that was before styler. I will double check.

Reply:
Solved by one-lining.